test/gui/shared/steps/steps.py

```diff
@@ -126,7 +126,7 @@ def step(context):
     startClient(context)
 
 
-@When(r'the user adds (the first|another) account with', regexp=True)
+@When(r'^the user adds (the first|another) account with$', regexp=True)
```

Comment:
why do we need this change?

Reply:
-When the user adds the first account with advanced configuration  
-When the user adds another account with

if we don't use a strict regex match then this two-step pointed to first step definition